frontend/public/components/masthead.jsx

```diff
@@ -19,8 +18,8 @@ export const getBrandingDetails = () => {
       productName = 'Red Hat OpenShift';
       break;
     case 'ocp':
-      logoImg = ocpLogoImg;
-      productName = 'Red Hat OpenShift Container Platform';
```

Comment:
this line change affected the disruption test detection.  I'm hoping others find this when they fix the test in openshift/origin master.